docs/docs/porting-part-of-a-html-site-to-gatsby.md

```diff
@@ -0,0 +1,569 @@
+  <Layout>
+    <h2>These are our staff:</h2>
+    <ul>
+      <li>
+        <Link to="/ella-arborist">Ella (Arborist)</Link>
+      </li>
+      <li>
+        <Link to="/sam-surgeon">Sam (Tree Surgeon)</Link>
+      </li>
+      <li>
+        <Link to="/marin-leafer">Marin (Leafer)</Link>
+      </li>
+    </ul>
+  </Layout>
+)
+```
+
+### Porting other section pages
+
+Now it's time for the work to really pay off! Ella's page is a matter of using your `<Layout>` component again and copying in the main content. Don't forget to change `class` to `className`! A `staffName` prop can be passed to `<Layout>` this time to change the dynamic page title, passing props is similar to an attribute on a HTML element:
+
+```jsx:title=/gatsby-site-section/src/pages/ella-arborist.js
+import React from "react"
+import Layout from "../components/Layout"
+import { Link } from "gatsby"
+
+export default () => (
+  {/* highlight-start */}
+  <Layout staffName="Ella">
+  {/* highlight-end */}
+    <h2>Ella is an excellent Arborist. We guarantee it.</h2>
+    <div className="bio-card">
+      <img
+        alt="Comically crude stick person sketch"
+        src="https://gatsby-html-partial-assets.s3.eu-west-2.amazonaws.com/person.png"
+      />
+      <p>Ella</p>
+    </div>
+  </Layout>
+)
+```
+
+The other 2 pages for Marin and Sam can now be made with a similar structure. Maybe you are even thinking about another component for the Bio Card!
+
+## Build
+
+With your new Gatsby application complete, it's time to integrate it into your existing HTML website. If you were to build the Gatsby application into static files now and upload them in the place of the existing HTML files, the paths of the links made by Gatsby would not be correct. There are a couple of Gatsby configuration options to fix this.
+
+### Path Prefix
+
+The `pathPrefix` option in `/gatsby-config.js` tells Gatsby the path at which the build output will be served from. Links to other pages and to the generated JavaScript files that provide [progressive enhancements](/docs/glossary/#progressive-enhancement) for the user will be prefixed with the `pathPrefix` value. This is the config addition for the example Gatsby application, served at the `/who` path:
+
+```js:title=/gatsby-config.js
+module.exports = {
+  plugins: [`gatsby-plugin-react-helmet`],
+  pathPrefix: `/who`, // highlight-line
+}
+```
+
+### Asset Prefix
+
+The `assetPrefix` option tells Gatsby about an external location, the asset domain, where its compiled JavaScript files will be served from. In addition to the `pathPrefix`, the link paths for Gatsby's generated JavaScript files will be prefixed with the `assetPrefix` value if provided. In the example website there are three assets already hosted at an asset domain; the style sheets `style.css` and `normalize.css`, and the image `person.png`. If Gatsby's generated JavaScript assets are also to be served from the asset domain, the `assetPrefix` must be set with its address:
```

Comment:
This seems like overkill to me -- I think it makes more sense to introduce global CSS so people can edit it directly, and we can skip the complexity for asset prefixes (unless it's needed for something specific related to porting within a project, like a pathPrefix is).